### ISSUE TITLE
[New App Shortcut]: 13News Now - WVEC

### ISSUE THREAD

**Issue by snagglewest** [PRProd/HA-Firemote#494]
### App Name

13News Now - WVEC

### Device Platform

Roku

### App Store Link and/or URL to .apk file if sideloaded

https://channelstore.roku.com/details/4c5f4473921cd3f828fa9ce069ae63ba/13news-now-wvec

### Required Information

media_content_type: app
app_id: "63399"
app_name: 13News Now - WVEC
source: 13News Now - WVEC
device_class: receiver
entity_picture: >-
  /api/media_player_proxy/media_player.roku_streaming_stick?
friendly_name: Roku Streaming Stick+
supported_features: 151481

### Optional Comments

Thank you in advance

**Comment by snagglewest** [comment]
Incredible card and I can't imagine how much time and effort you've put into it!

Just thinking out loud here but it seems like a LOT of continuing work to add each a app as folks request them.  At least for the Roku devices its easy to get the app ID.  (http://[deviceip]:8060/query/apps)  Maybe add an option to  add an app ID and a path to a user added icon?   I don't know how the firestick etc work but if it cuts down your Roko "todo" list it might be worth while.  Seems simple but I'm sure the underlying code is a little more complicated than I think :)

**Comment by PRProd** [comment]
Greetings @snagglewest!

YOU'RE IN LUCK!  I actually DID create a way for users to create their own buttons, AND include their own artwork for them!  In the [App/Function Launcher Buttons](https://github.com/PRProd/HA-Firemote/wiki#appfunction-launcher-buttons) section of the Firemote wiki, there is a section called [Create a Custom Button for Yourself](https://github.com/PRProd/HA-Firemote/wiki#create-a-custom-button-for-yourself) which details how to do it!

I don't have any examples of Roku in there yet, but I'm more than happy to add one if you'd find it helpful!

**Comment by snagglewest** [comment]
I found the custom button section but I'm no programmer, just a cut-n-paster, so I gave up.  A Roku example would really be helpful so I don't have to bug you for all our local channel apps :)

**Comment by PRProd** [comment]
I added a Roku section to that help article for you to look at.

Here is an example of some YAML you could use to make a button for this request, and put it in your app launcher 2 position:
```yaml
custom_launchers:
  - friendly_name: 13News Now
    label: News
    color: white
    background: blue
    action: media_player.select_source
    target:
      entity_id: {YOUR ROKU MEDIA PLAYER ENTITY GOES HERE}
    data:
      source: 13News Now - WVEC
app_launch_2: customlauncher 13News Now
```

**Comment by PRProd** [comment]
Good news @snagglewest!

I have just published Firemote version [v4.0.1](https://github.com/PRProd/HA-Firemote/releases/tag/v4.0.1) which contains your app shortcut request.  Enjoy!

-Doug
<hr>

After updating to the newest version, if you are unable to see the new updates on your Firemote card, visit [this link](https://github.com/PRProd/HA-Firemote/wiki/Force-a-Refresh) for help.
